src/components/IdeaCard/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Just wondering, will we allow editing of the idea? Or is what the overlay is for?

Reply:
I didnt think about editing, overlay is just to show a full description and full info about members. 
But of course we can let people edit an idea. But then it might me not very safe, if anyone can edit. What do u think?

Reply:
That's true, we would have to authenticate the person... it's getting too complicated. But what if they do something wrong? I'd say this is "nice to have", but not a must right now

Reply:
Then lets do it without for now. And if we have some time in the end, we can add editing. It shouldn't be complicated